### PR TITLE
CU-85zta452d - Support releasing multiple OSS projects

### PR DESCRIPTION
**Pull request by undera**
Allows releasing both helm-dashboard and komoplane

### CODE REVIEW

File: .buildkite/pipeline.yml
Comment (MickaelAlliel) [inline .buildkite/pipeline.yml:6]: @undera You'll need to update https://github.com/komodorio/helm-dashboard/blob/main/.github/workflows/publish-chart.yaml#L43 this line as well to include `OSS` in the commit message of helm-dashboard to not break it's own CI

**Review by MickaelAlliel — CHANGES_REQUESTED** [review]
See comment - tell me when change is ready in helm-dashboard so we can merge this one

File: .buildkite/pipeline.yml
Comment (undera) [reply .buildkite/pipeline.yml:6]: This is exactly what I've prepared, and I'm waiting for this PR to be merged, to followup with change in helm-dashboard repo. Do you want me to do the change before this PR merges?

Comment (undera) [reply .buildkite/pipeline.yml:6]: @MickaelAlliel anyway, here's corresponding PR: https://github.com/komodorio/helm-dashboard/pull/369

**Comment by undera** [comment]
@MickaelAlliel I have made the change the helm-dashboard repo

**Review by MickaelAlliel — APPROVED** [review]

File: .buildkite/pipeline.yml
Comment (MickaelAlliel) [reply .buildkite/pipeline.yml:6]: Didn't think too much about the order I'll be honest.
But since you've already merged the helm-dashboard one, let's merge this one now :)

**Pull request merged**
